\pdfminorversion=5
%\documentclass[english,a4paper]{article}
%\documentclass[english,a4paper]{scrartcl}
%\documentclass[english]{sig-alternate}
\documentclass[english,11pt,a4paper]{scrartcl}
\usepackage[utf8]{inputenc}
\usepackage{lmodern}
\usepackage[T1]{fontenc}
\usepackage{microtype}
\usepackage{amssymb,xfrac}
\usepackage{appendix}
%\usepackage{authblk}
%\usepackage[DIV=10]{typearea}
\usepackage{amsfonts, amsmath, hanging, natbib, parskip}
\usepackage{babel}
\usepackage{hyperref}
\usepackage{graphicx}
\usepackage{xcolor}
\definecolor{dark-red}{rgb}{0.4,0.15,0.15}
\definecolor{dark-blue}{rgb}{0.15,0.15,0.4}
\definecolor{medium-blue}{rgb}{0,0,0.5}
\hypersetup{
    colorlinks, linkcolor={dark-red},
    citecolor={dark-blue}, urlcolor={medium-blue}
}
\begin{document}
\title{Pilot study on genomic predictions for fruit traits in peach 
(\emph{Prunus persica}) populations}
%\author{F. Biscarini,\and N. Nazzicari, \and D. Bassi, \and I. Pacheco
%  Cruz, \and A. Stella, \and L. Rossini, \and D. Gianola}
%\author[1]{F. Biscarini\thanks{A.A@university.edu}}
%\author[1]{N. Nazzicari}
%\author[2]{D. Bassi}
%\author[2]{I. Pacheco Cruz}
%\author[1]{A. Stella}
%\author[3]{L. Rossini}
%\author[4]{D. Gianola}
%\affil[1]{Department of Bioinformatics, PTP}
%\affil[2]{Department of Plant Biology, University of Milan}
%\affil[2]{Department of Plant Genomics, PTP}
%\affil[2]{Department of Animal Breeding \& Genetics, University of Wisconsin}

%\renewcommand\Authands{ and }
\date{}
\maketitle

\section*{Introduction}

Pilot study on genomic predictions for fruit weight and fruit quality traits (brix and acidity) in a collection of European peach tree populations.

\section*{Material \& Methods}

\subsection*{Plant material}

CRA: IF7310828 x (IF7310828xFerganensis), Backcross of IF7310828 x (IF7310828xFerganensis)  [840]

INRA-Av: (SD40 x Summergrand)x(ZÃ©phyr) [1441], (Bolinha)x(Bolinha) [111], (Summergrand)x(P davidiana) [11 to be excluded]. We'll use data from INRA-AV-original-codes.

IRTA: Belbinette x Nectalady [331], BigTop x ArmKing [294], BigTop x Nectacross [166]. 423 NULL (field "cross") to be investigated.

UMIL: BxO [305], MxR [144], WxBy [478].

Not all individual trees have data on all phenotypic traits.


\subsection*{Phenotypes}

DB mySql pesco (in localhost/phpmyadmin)
also in peach_gs.Rproj (Nelson)

\subsection*{Genotypes}

file allSNPs.ped: 3297 individuals 

map file: RosBREED_Peach_10k_11494376_B.bpm.map

\subsection*{Mendelian errors and NULL alleles}

\subsection*{Genomic relationships between individual trees}

%\input{TableStats}


\subsection*{G-BLUP}
A GBLUP of the following form was run to estimate the genomic breeding
values for average fruit weight:

\begin{equation}
\boldsymbol{y}=\boldsymbol{1\mu}+\boldsymbol{Zu}+\boldsymbol{e}
\end{equation}

where $\boldsymbol{y}$ is the vector of the average fruit weights in
grams, $\mu$ is the overall mean, $\boldsymbol{u}$ is the
vector of the random genomic effects of each treee, and $\boldsymbol{e}$ is a
vector of residuals. $\boldsymbol{1}$ is a vector of 1s and
$\boldsymbol{Z}$ is a matrix relating observations in $\boldsymbol{y}$ to
the random genomic effects in $\boldsymbol{u}$.

For the random part of the model, the genetic variance is
$Var(u)=G\sigma^2_u$, and the residual variance is
$Var(e)=R=I\sigma^2_e$.
For the genetic variance either the matrix of genomic relationships
\`{a} la Van Raden (GVR) or that \`{a} la Astle \& Balding (GAB) were used. 

The resulting mixed model equations looked therefore like:

\begin{equation}
\begin{bmatrix}
\mu \\
\boldsymbol{\hat{u}}
\end{bmatrix}
=
\begin{bmatrix}
\boldsymbol{1^\prime}\boldsymbol{1} & \boldsymbol{1^\prime}\boldsymbol{Z} \\
\boldsymbol{Z^\prime}\boldsymbol{1} & \boldsymbol{Z^\prime}\boldsymbol{Z}+\boldsymbol{G^{-1}\alpha}
\end{bmatrix}
\begin{bmatrix}
\boldsymbol{1^\prime}\boldsymbol{y} \\
\boldsymbol{Z^\prime}\boldsymbol{y}
\end{bmatrix}
\end{equation}

where $\alpha$ is the ration between the residual and the genetic
variance ($\frac{\sigma^2_e}{\sigma^2_u}$).

A 5-fold cross validation scheme was employed to estimate the predictive
ability of the two different implementations of \emph{GBLUP}. The total
dataset was split into 5 subsets: 4 of these were used as training set
to set up the model and estimate variance components; the remaining $1/5$ of the data was
used as validation in which the known observations (the \emph{y's} were
masked and artificially considered to be missing. Each of the 5 subsets
was used in turn as validation set, to make sure that every observation
was at least once predicted from the model.
This process was repeated 100 times so to have -in the end- 500
replicates of the analysis.
 
The predictive ability was defined as the Pearson correlation between
the genomic breeding values (the $\hat{u}$s of
the model) and the known -masked- average fruit weights of each
tree in the validation set.

For each of the 500 replicates, the heritability of the trait (average
weight of 10 fruits per tree) was estimated as the ratio between the
genetic variance and the total phenotypic variance ($\frac{\sigma^2_u}{\sigma^2_u+\sigma^2_e=\sigma^2_p}$).

The mean predictive ability over the 500 replicates, its standard
deviation, the mean $h^2$ and its standard errors are reported in
Table~\ref{tab:CrossValGBLUP}.  

Such high heritability ($\sim 80\%$), is likely to be due to the fact
that we are analysing average measurements. Let the residual variance of
a single item be $\sigma^2_e$, then the residual variance of the mean of
n items would be $\sfrac{\sigma^2_e}{n}$, and the heritability: 
\begin{equation} \label{eq:h2mean}
h^{2*}=\frac{\sigma^2_u}{\sigma^2_u+\sfrac{\sigma^2_e}{n}}=\frac{n}{n+\sfrac{(1-h^2)}{h^2}}
\end{equation}

where $h^{2*}$ is the heritability of average observations and $h^2$ is
the heritability of single observations. This implies that if the
heritability of the average weight of 10 fruits is \textasciitilde 0.8,
the heritability of the weight of a single fruit would be (from
rearrangements of equation~\ref{eq:h2mean})
\textasciitilde 0.3.

\begin{table}
%\begin{center}
\centering
\begin{tabular}{l c c c c}
G matrix & $\overline{PredAb}$ & $\sigma_{PredAb}$ & $h^2$ & s.e. \\
\hline
GVR & 0.727 & 0.084 & 0.77 & 0.077 \\
GAB & 0.730 & 0.082 & 0.79 & 0.074 \\
\end{tabular}
\caption{Predictive ability and heritability for average fruit weight in
  peach trees (results are averages over 500 cross-validation replicates).}
\label{tab:CrossValGBLUP}
%\end{center}
\end{table}

\subsection*{Repeatability model}
Repeated observations on average fruit weight in two consecutive years
(2007 and 2008) are available for the peach tree population under
analysis. Therefore, it might be appropriate to use this additional
information by fitting a repeatability model to the data.
In a repeatability model the phenotypic variance is partitioned into
the genetic variance (we are now considering just the additive genetic
effects), permanent
environmental variance and residual (or, loosely speaking,
temporary environmental) variance: $\sigma^2_y=\sigma^2_a+\sigma^2_{pe}+\sigma^2_e$.

A repeatability model allows for the estimation not only of the
\emph{heritability} ($h^2=\frac{\sigma^2_a}{\sigma^2_y}$), but also of the
correlation between successive records of an individual, known as
\emph{repeatability} ($r=\frac{\sigma^2_a+\sigma^2_{pe}}{\sigma^2_y}$). 

In matrix notation, our repeatability model had the following form:

\begin{equation}
\boldsymbol{y}=\boldsymbol{1\mu}+\boldsymbol{Za}+\boldsymbol{Wpe}+\boldsymbol{e}
\end{equation}

where
{\renewcommand\labelitemi{}
\begin{itemize}
\item $\boldsymbol{y}$ is the vector of observations (average weight of 10
  fruits per tree per year)
\item $\boldsymbol{\mu}$ is the overall mean
\item $\boldsymbol{a}$ is the vector of individual genomic values
\item $\boldsymbol{pe}$ is the vector of permanent environmental effects
  (one per tree)
\item $\boldsymbol{1}$ is a vector of 1s
\item $\boldsymbol{Z}$ and $\boldsymbol{W}$ are incidence matrices relating
  records to the genomic and permanent environment effects
\end{itemize}
}



\begin{table}
%\begin{center}
\centering
\begin{tabular}{l c c c c}
Genetic Parameter & estimate & s.d. & std error & s.d. \\
\hline
Repeatability & 0.822 & 0.020 & 0.044 & 0.004 \\
Heritability & 0.774 & 0.173 & 0.047 & 0.009 \\
\end{tabular}
\caption{Heritability and repeatability estimates, and their standard
  error as estimated by ASREML (+ respective standard
  deviations) for average peach weight (results are averaged over 500 cross-validation replicates).}
\label{tab:repeatability}
%\end{center}
\end{table}


\pagebreak

\bibliographystyle{plainnat}
\bibliography{biblio}

\end{document}

